test/compiler/js/style.js

```diff
@@ -1,7 +1,6 @@
 riot.tag('style-test', '<h3>{ opts.title }</h3>', 'style-test { display: block; border: 2px }', function(opts) {
-    this.foo = function() {
-
-        this.update()
-    }.bind(this);
+  this.foo = function() {
```

Comment:
@tipiirai You should write up some contribution guidelines. 

I'm a pretty big believer in super focused PRs and maintaining a super clean git history. I also know that a LOT of people don't care about that kind of stuff. @GianlucaGuarini is doing hero's work here so a few guidelines would help us out a ton.